src/blocks/sound/pulseaudio.rs

```diff
@@ -238,11 +236,11 @@ impl Client {
                 }
 
                 loop {
+                    use std::sync::mpsc::TryRecvError;
```

Comment:
Should this be at the top of the file? Is there any impact of having it in the body (of a loop)?

Reply:
`tokio::sync::mpsc::error::TryRecvError` has the same name and we use both std and tokio channels in this file, so I thought minimizing the scope of this `use` might be a good idea.